test/miniml.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
minimal.py? or is miniml.py intended?

Reply:
It is a mock name for a framework. Do you prefer mini_ml ? Or mocked_framework ?

Reply:
ah, mini ml lol that works